Formula/p/pivit.rb

```diff
@@ -0,0 +1,26 @@
+class Pivit < Formula
+  desc "Sign and verify data using hardware (Yubikey) backed x509 certificates (PIV)"
+  homepage "https://github.com/cashapp/pivit"
+  url "https://github.com/cashapp/pivit/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "67a2f353c820aad9c9a4cc937ba256c75cb36044baa0578b74a1bfbdf4e6bd90"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  on_linux do
+    depends_on "pcsc-lite"
+  end
+
+  def install
+    system "make", "pivit"
+    bin.install "pivit"
```

Comment:
Will a `make install` not do that?

Reply:
This tool is written in Go, and so `make install` will call `go install` which places the binaries in `$GOBIN`.
Unfortunately, there's no "good" way to set the destination directory when executing `go install`.

I think a more explicit procedure that builds the binary and then copies it to the right cellar, is clear and easy enough to maintain.
WDYT?